apps/meteor/app/dolphin/server/lib.ts

```diff
@@ -22,6 +23,7 @@ const config = {
 const Dolphin = new CustomOAuth('dolphin', config);
 
 function DolphinOnCreateUser(options: any, user?: IUser) {
+	// TODO: callbacks Fix this
```

Comment:
Is this TODO being tracked in a task? Can you create a technical debt ticket to address it if you're not going to do it in this PR?

Reply:
if i had created a task i would not have put a todo in the code. as you can see i prefixed it with 'callbacks' this will be handled in the next tasks related to callbacks